### ISSUE TITLE
semantic is not yet checked

### ISSUE THREAD

**Issue by CrazyCrud** [coneda/kor#293]
First of all, thank you very much for the work on _kor_.
I followed the install instructions, but when I open the site the following error gets thrown:
```
https://github.com/jlindsey/semantic (at master@1bb0684) is not yet checked out. Run `bundle install` first. (Bundler::GitError)
<pre>  /home/kor/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/bundler-2.1.4/lib/bundler/source/git.rb:204:in `rescue in load_spec_files&#39;
```

I also re-ran the command ```gem install bundler``` but the package still seems missing.

I'm on a Ubuntu 18.04 machine.

Thank you in advance
Best regards

**Comment by moritzschepp** [comment]
Hey @CrazyCrud, thanks for the report!

The error message complains about a missing gem (dependency). All required gems should have been installed with the command

    bundle install --without development test

Did that throw any errors?

**Comment by CrazyCrud** [comment]
Thank you @moritzschepp for your quick response.
Sorry, I meant that I executed ```bundle install --without development test```.
The output of the installation was the following:
```
Bundle complete! 40 Gemfile dependencies, 68 gems now installed.
Gems in the groups development and test were not installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
```
No error is thrown and the gem _semantic 1.5.0 from https://github.com/jlindsey/semantic (at master@1bb0684)_ is listed too.
If I run ```bundle info semantic``` it says:
```
Summary: Semantic Version utility class
        Homepage: https://github.com/jlindsey/semantic
        Path: /home/kor/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/bundler/gems/semantic-1bb06848c42e
```
But the exception is still the following:
```
<p>-------- The exception is as follows: -------</p>
https://github.com/jlindsey/semantic (at master@1bb0684) is not yet checked out. Run `bundle install` first. (Bundler::GitError)
<pre>  /home/kor/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/bundler-2.1.4/lib/bundler/source/git.rb:204:in `rescue in load_spec_files&#39;
```
Only this gem seems to be missing.

Best regards

**Comment by moritzschepp** [comment]
Oh I see. The semantic gem is pulled directly from github before all other dependencies. I think its simply the first gem it checks and all others are probably not available either. I suspect that apache is using a different ruby version than your shell. If you type `rbenv versions` in the shell it should tell you which ruby is active. "system" is the ruby installed by apt-get. You can set a ruby version as a global default with `rbenv global 2.5.8` and apache will use it as well. If you need to set a different version for apache, use `echo "2.5.8" > /path/to/kor/directory/.ruby_version`.

Does that work?

BTW, the install manual was written when ruby 2.4 was still maintained. Perhaps you want to use 2.5.8 instead, which works as well. I will update the manual shortly.

**Comment by CrazyCrud** [comment]
@moritzschepp Thank you very much for your response and sorry for the long wait. 
I set the global version to 2.4 via `rbenv global` with which I also installed the `gem bundler`. I also tried to set the version for apache but I still get the same error. But I think you are totally right and there is a version problem. I will try to spend some more time on it and figure it out and let you know.

Again, thank you very much for your help!

**Comment by moritzschepp** [comment]
Can you post your passenger related apache config files? There are usually two of them.

**Comment by CrazyCrud** [comment]
This is the content of the `/etc/apache2/mods-available/passenger.conf` file:
```
<IfModule mod_passenger.c>
  PassengerRoot /usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini
  PassengerDefaultRuby /home/kor/.rbenv/shims/ruby
</IfModule>
```

The Ruby version is the following:
```
rbenv versions
* system (set by /home/kor/.rbenv/version)
  2.4.4
  2.5.1
```

     
Best regards

**Comment by moritzschepp** [comment]
The passenger config looks good. I think your `rbenv global 2.4.4` didn't work, though: rbenv shows it is using "system" globally. Is there any feedback if you run that command?

**Comment by CrazyCrud** [comment]
Sorry, you're right, unfortunaley the version got reset to the system version, the server was restarted.

I ran ```rbenv global 2.4.4``` and the version is correctly set:
```
  system
* 2.4.4 (set by /home/kor/.rbenv/version)
  2.5.1
```
```
2.4.4 (set by RBENV_VERSION environment variable)
```
I installed the bundler again:
```
rbenv shell 2.4.4
gem install bundler
```
It got installed successfully:
```
Successfully installed bundler-2.1.4
```
I already did all the remaining steps (cloning, database creation,...) and after I called the site it worked!

Thank you very much for your help! I hope there will be no problem with the Ruby version again.


Best regards & thank you again
